synapse/replication/http/_base.py

```diff
@@ -202,6 +227,9 @@ async def send_request(instance_name="master", **kwargs):
                 # the master, and so whether we should clean up or not.
                 while True:
                     headers = {}  # type: Dict[bytes, List[bytes]]
+                    # Add an authorization header, if configured.
+                    if replication_secret:
+                        headers[b"Authorization"] = [b"Bearer " + replication_secret]
```

Comment:
I wasn't sure whether to use `Bearer` or `Basic` here, although it doesn't really matter since we just care about the shared secret part.

Reply:
I think `Basic` requires a certain format for the token, so `Bearer` is probably the right thing here